apps/frontend/src/components/review/ProposalReviewContent.tsx

```diff
@@ -121,17 +116,18 @@ const ProposalReviewContent = ({
         technicalReview?.technicalReviewAssigneeId === user.id;
 
       return isUserOfficer ||
-        canEditAsFapChairOrSec ||
+        isFapSec ||
+        canEditAsFapSec ||
         canEditAsInstrumentSci ||
         isInternalReviewer ? (
         <Fragment key={technicalReview.id}>
-          {!!technicalReview && (
+          {!!technicalReview && !isFapSec && (

```

Comment:
Why is this competent only seen if you're not a FapSec? Sorry if I am miss understanding

Reply:
Same apply for the other !FapSecs below this

Reply:
> Why is this competent only seen if you're not a FapSec? Sorry if I am miss understanding

That wouldve been a misunderstanding on my behalf as I didnt know that FAP secs were allowed to edit and set tech reviews. Thank you for clarifying that!